typespec-tests/tsp/enum.tsp

```diff
@@ -70,7 +71,7 @@ model Operation {
   priorityValue: Priority.Low;
   colorValue: Color.Green;
   colorModelValue: ColorModel.Blue;
-  unitValue: Unit.Milligram;
+  unitValue?: Unit.Milligram;
```

Comment:
Without this PR, client would generate a `Milligram` enum. To me this does not look OK.